.pre-commit-config.yaml

```diff
@@ -1,24 +0,0 @@
-repos:
-
-- repo: https://github.com/asottile/seed-isort-config
-  rev: v1.7.0
-  hooks:
-  - id: seed-isort-config
-- repo: https://github.com/pre-commit/mirrors-isort
-  rev: v4.3.20
-  hooks:
-  - id: isort
-    additional_dependencies: [toml]
-- repo: https://github.com/ambv/black
-  rev: stable
-  hooks:
-  - id: black
-- repo: https://github.com/pre-commit/mirrors-pylint
-  rev: v2.3.1
-  hooks:
-  - id: pylint
-    additional_dependencies: [toml]
-    name: PyLint
-    entry: pylint
-    args: [--rcfile=.pylintrc, --exit-zero]
-    verbose: true
```

Comment:
Why remove this file? :|

Reply:
Because the same stuff are already done by the action that act on push and pull request event. I think that file is not necessary anymore. Do you think we should maintain that?

Reply:
Wait what? I fire a pull request and GitHub changes my commits? I'm strongly against this.

And if it's just a check, fine, but don't remove the integration with my editor.

Reply:
> Wait what? I fire a pull request and GitHub changes my commits? I'm strongly against this.

No changes in commits.

> And if it's just a check, fine, but don't remove the integration with my editor.

Sounds fair